notebooks/2_feature_engineering.ipynb

```diff
@@ -189,77 +203,134 @@
     "\n",
     "print(documents)\n",
     "print(vecs)"
-   ]
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "* In this example, we show different embeddings from simple `TFIDF` to more complex `BERT`. \n",
     "* Notice how the `Embedding` can be followed with one more transformation operations, such as `NMF` or `SVD`. \n",
     "* In general, the `Transformation` reduces the dimensionality of the text representation to create succint embeddings. \n",
     "* Running `fit_transform` on the documents return the embedding of each document. \n",
     "* Checkout different word options, pre-trained models, and other transformations.\n",
     "* See the rich list of [TextWiser Embeddings](https://github.com/fidelity/textwiser#available-embeddings)."
-   ]
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "# Sequential Data via Seq2Pat\n",
     "\n",
-    "* Time horizon is another source of information to build advanced recommenders.\n",
+    "* For most applications, structured data for user features via Selective and unstructured data for item features via TextWiser will suffice.\n",
```

Comment:
I reworded this a little bit based on feedback in User Group Chat. Most people would not need seq2pat in their MVP1 to get a mab2rec model, but the notebook does not say that. Some folks tripped over seq2pat because it does not compile due to C++ backends etc.. and then people spend time trying to fix that issue.. whereas, they would be completely fine if they jumped over this section, in their first end-to-end run. They can of course come back to feature engineering to incorporate s2pat features later on, but it is overkill in the first try and blocks people.

Reply:
Makes sense to me. I also would like to append the C++ compiler requirement to the last bullet. 

"Let's install the library and explore a quick start example. Notice that Seq2Pat requires a C++ compiler to build the backend module. Detailed installation requirments can be found [here](https://fidelity.github.io/seq2pat/installation.html)."

Reply:
good poiint!